azure/init.sh

```diff
@@ -43,6 +45,17 @@ rsync -aXS /tmp/home_copy/. /home/${TARGET_USER}/.
 # We do not want the password prompt, because we want to run tests without any user input
 echo '${TARGET_USER}     ALL=(ALL) NOPASSWD:ALL' >>/etc/sudoers
 
+# we will use port 3456 to not hit security rule 103
+echo 'Port 3456' >> /etc/ssh/sshd_config
+echo 'Port 22' >> /etc/ssh/sshd_config
+
+# necessary for semanage, VMs have secure linux
+yum install -y policycoreutils-python
+# we need to enable the new port from semanage
```

Comment:
[ let me do some research for what is semanage :) ]

Reply:
we have secure linux and in secure linux, we need to do some extra commands to enable some security related stuff. semanage helps with adding the port in this case